packages/core/src/sdk.ts

```diff
@@ -20,12 +21,28 @@ export interface ClientClass<F extends Client, O extends Options> {
 export function initAndBind<F extends Client, O extends Options>(
   clientClass: ClientClass<F, O>,
   options: O,
+  defaultIntegrations: Integration[] = [],
 ): void {
   if (shimGetCurrentClient()) {
     return;
   }
 
   const client = new clientClass(options);
   client.install();
+
+  let integrations = [...defaultIntegrations];
+  if (Array.isArray(options.integrations)) {
```

Comment:
nit: `Array.isArray` is only supported by IE 9 + right? We should just keep that in mind.